### PR TITLE
performance test - support to customize workload size

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: test/performance/README.md [hunk 1]
```diff
@@ -17,6 +18,55 @@ We can use the `perftool` to do the performance test for the multicluster contro
...
+      metadata:
+        name:   <cluster-name>-work-<work-number>
+        namespace: default
+      data:
+        test: "I'm a test configmap"
+```
+
+You can customize your own workload by following steps
+
+1. Define your manifestworks in a directory and use `` annotation to specify your expected manifestwork counts, e.g.
```
Comment (clyang82) [inline test/performance/README.md:42]: missed `perftest.open-cluster-management.io/expected-work-count`

File: test/performance/README.md [hunk 2]
```diff
@@ -17,6 +18,55 @@ We can use the `perftool` to do the performance test for the multicluster contro
...
+apiVersion: work.open-cluster-management.io/v1
+kind: ManifestWork
+metadata:
+  name: work-1
+  annotations:
+    "perftest.open-cluster-management.io/expected-work-count": "10"
+spec:	
+  workload:
+    manifests: <your-manifests>
+
```
Comment (clyang82) [inline test/performance/README.md:54]: you may need `---` to separate the yaml file

File: test/performance/README.md [hunk 3]
```diff
@@ -17,6 +18,55 @@ We can use the `perftool` to do the performance test for the multicluster contro
...
+  annotations:
+    "perftest.open-cluster-management.io/expected-work-count": "10"
+spec:	
+  workload:
+    manifests: <your-manifests>
+```
+
+2. Use `--work-template-dir` flag to specify your directory path
+
+Then the `perftool` will wrap your manifestworks with `configmap` to simulate the same size with your customized workloads.
```
Comment (clyang82) [inline test/performance/README.md:68]: wrap the above manifestwork in a configmap? or wrap configmap with manifestwork?
what does `expected-work-count` mean?

Comment (skeeey) [reply test/performance/README.md]: make this part description more clear

File: test/performance/README.md (hunk 2 above)
Comment (skeeey) [reply test/performance/README.md:54]: added

File: test/performance/README.md (hunk 1 above)
Comment (skeeey) [reply test/performance/README.md]: added

**Pull request merged**